extensions/ql-vscode/src/vscode-tests/jest-runner-vscode.config.base.js

```diff
@@ -11,7 +11,6 @@ const config = {
   launchArgs: [
     "--disable-gpu",
     "--extensions-dir=" + path.join(rootDir, ".vscode-test", "extensions"),
```

Comment:
Should we be disabling potentially conflicting extensions here as well? There was a list of them when running with mocha. Not sure if these extensions are being explicitly disabled any more.

Reply:
The extensions are being disabled in the configuration file specific to the `cli-integration` tests:

https://github.com/github/vscode-codeql/blob/7ba90275a8c71f7e06142035f07cc762af7765fb/extensions/ql-vscode/src/vscode-tests/cli-integration/jest-runner-vscode.config.js#L14-L19

We are also disabling all extensions in the configuration for the `no-workspace` and `minimal-workspace` tests:

https://github.com/github/vscode-codeql/blob/7ba90275a8c71f7e06142035f07cc762af7765fb/extensions/ql-vscode/src/vscode-tests/no-workspace/jest-runner-vscode.config.js#L6

This should match the previous Mocha setup in terms of which extensions are enabled/disabled.